### DISCUSSION TITLE
Logo/Icon?

### DISCUSSION THREAD

**Discussion by Crewski** [tagyoureit/nodejs-poolController#671] [General]
Does nodejs-PoolController happen to have a logo or icon?  I'm looking to add one to my Home Assistant integration so I figured I'd check to see if there is an "official" one.  Thanks.

**Comment by rstrouse** [comment]
There isn't really one.  The beach ball is displayed on the dashPanel tabs.  Any suggestions from you artsy types?

**Reply by gw8674** [reply]
After a little recovery from last night, I started working on gathering tax info **ugh**
Thought I would play around with something a bit more on the "lighter side" instead.
Now back to the 'real" world. 
Happy New Year all...
![njspC](https://user-images.githubusercontent.com/52094163/210186883-87a2be08-c91c-4e9f-a38c-67ddeb8b401c.png)

**Comment by Crewski** [comment]
I'm not artsy by any means...in fact the exact opposite.
![icon](https://user-images.githubusercontent.com/8012153/217299060-2701f8eb-44c9-432e-a3d4-60d156287b49.png)
![logo](https://user-images.githubusercontent.com/8012153/217299064-8c02f2d2-f759-4cbd-bfc8-150f444db864.png)

**Reply by rstrouse** [reply]
Funny, I just included some on my pull request.  I messed around with some of the ones with a long wide side but when you scale them to the 32x32 range for icon then tend to look like a line.  Perhaps we can figure some combination of our work to get to something close to square.

**Comment by Crewski** [comment]
What about the nodejs logo with a PC inside?  That should scale down decently for the icon.  The logo could be that with just "nodejs-poolController" after it.

![icon v2](https://user-images.githubusercontent.com/8012153/217530038-40cdd9ed-7fd6-4e9a-aba9-eee22b8a36f8.png)
![logo v2](https://user-images.githubusercontent.com/8012153/217530040-5c6943a3-b8d8-4ffe-aaa1-3b3c2a5a596d.png)
